Makefile

```diff
@@ -330,6 +331,7 @@ Tools/quabs: Tools/quabs-git/release/quabs
 
 Tools/quabs-git/release/quabs: Tools/quabs-git
 	cd Tools/quabs-git; mkdir -p release
+	sed -i.bak 's/GIT_TAG 5.6.8/GIT_TAG 5.7.0/' Tools/quabs-git/libsolve/src/CMakeLists.txt
```

Comment:
I updated our libsolve and quabs forks.  
Please verify that it now works without this line.